catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -102,7 +102,7 @@
 
   private Set<String> rootResourceDirectories = new HashSet<>();
 
-  private boolean followRedirects = false;
+  private boolean followRedirects = true;
```

Comment:
I see that there is already a note about this in the documentation:
https://github.com/codice/ddf/blob/080dea730aa7c06240cb97e74cc5b43203242242/distribution/docs/src/main/resources/content/_sources/confluence-federated-source.adoc#L52
Is this the intended behavior? Is the code change necessary?

✏️ Would it be easy to add/update a unit test for this change?
❗️ The documentation needs to be updated:
https://github.com/codice/ddf/blob/080dea730aa7c06240cb97e74cc5b43203242242/distribution/docs/src/main/resources/content/_tables/URLResourceReader.adoc#L18-L22

Reply:
@stustison said he couldn't remember if there was a good reason, security-wise, to disallow it. Maybe if the redirect URL was leading to a harmful site? I'm not sure, but it seems that if it's a Confluence source then it should be ok. So the change would just improve functionality by not even running into that download failure in the first place.

Reply:
I can look into unit tests for the change since I don't think there were any specifically for redirects. And right, I can make the update for the documentation.

Reply:
I couldn't remember why we were disallowing redirects and anything I could think of was already mitigated by the fact that we're now using our secure client in the resource reader. I asked @pklinef as well and he couldn't remember either.

Reply:
Thanks for checking! I'll ✅ once the documentation is updated.

Reply:
Is there a good reason to keep the `followRedirects` option around, or can it be removed?

Reply:
Updated docs. Also, @jhunzik has a patch for configurations that prevents the downloads from failing so waiting for that to see if it'll change things with that option.

Reply:
This line still needs to be updated even after @jhunzik's patch, right?
https://github.com/codice/ddf/blob/080dea730aa7c06240cb97e74cc5b43203242242/distribution/docs/src/main/resources/content/_sources/confluence-federated-source.adoc#L52